conf/application.conf

```diff
@@ -65,6 +65,13 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuditModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
 
+
+play.filters {
```

Comment:
upscan confirms to the callback url that a file has been processed, issue is that they don't send a csrf token so we need to add an exception for them